diff --git a/node_modules/@myst-theme/site/src/components/DocumentOutline.tsx b/node_modules/@myst-theme/site/src/components/DocumentOutline.tsx
index 7131501..45f7a43 100644
--- a/node_modules/@myst-theme/site/src/components/DocumentOutline.tsx
+++ b/node_modules/@myst-theme/site/src/components/DocumentOutline.tsx
@@ -126,5 +126,5 @@
 function useMutationObserver(
-  targetRef: RefObject<Element>,
+  targetRef: RefObject<Element | null>,
   callback: MutationCallback,
   options: Record<string, any>,
 ) {
